@xen-orchestra/web-core/lib/components/UiLegend.vue

```diff
@@ -0,0 +1,97 @@
+<!-- v1.0 -->
+<template>
+  <li class="legend">
+    <span class="circle" :class="circleColorClass"></span><span class="label typo p3-regular">{{ label }}</span
+    ><UiIcon
+      v-tooltip="{
+        placement: 'top',
+        content: tooltip ? $t(tooltip) : '',
+      }"
+      :icon="tooltip ? faCircleInfo : undefined"
+      color="info"
+      class="ui-icon"
+    />
+
+    <span class="value-and-unit typo c3-semi-bold">{{ value }} {{ unit }}</span>
+  </li>
+</template>
+
+<script setup lang="ts">
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { computed } from 'vue'
+
+const props = defineProps<{
+  value?: number
+  state: 'success' | 'warning' | 'error' | 'disabled' | 'dark-blue' | 'default'
+  label: string
+  unit?: string
+  tooltip?: string
+}>()
```

Comment:
- Rename `state` to `color` to be more aligned to DS
- Extract color type to `LegendColor` in `types/color.type.ts`
- Keep required props together
- Remove `label` prop in favor of a slot

```suggestion
defineProps<{
  color: LegendColor
  value?: number
  unit?: string
  tooltip?: string
}>()
```